gui/webdisplay/src/RWebWindow.cxx

```diff
@@ -443,7 +443,10 @@ void RWebWindow::ProvideQueueEntry(unsigned connid, EQueueEntryKind kind, std::s
       fInputQueue.emplace(connid, kind, std::move(arg));
    }
 
-   InvokeCallbacks();
+   // if special python mode is used, process events called from special thread
+   // there is no other way to get regular calls in main python thread,
+   // therefore invoke widgets callbacks directly - which potentially can be dangerous
```

Comment:
Why is this potentially dangerous?

Reply:
While `canvas.Update()` or any other canvas method can be called from python execution thread, 
but at the same time callbacks can be processed from special thread where `gSystem->ProcessEvents()` are invoked.

Reply:
It is same situation with normal ROOT graphics as well - nothing new.